frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -226,20 +237,23 @@ function BlankBallotWarningScreen(): JSX.Element {
 
 function OtherReasonWarningScreen(): JSX.Element {
   const [confirmTabulate, setConfirmTabulate] = useState(false);
-
   return (
     <ScreenMainCenterChild infoBar={false}>
       <ExclamationTriangle />
       <CenteredLargeProse>
-        <h1>Review Your Ballot</h1>
+        <h1>Can’t Interpret Ballot</h1>
+        <p>
+          The ballot scanner was unable to determine if the scanned document is
+          a ballot.
+        </p>
         <p>
           <Button primary large onPress={scanner.returnBallot}>
-            Return my ballot
+            Return Ballot
           </Button>
         </p>
         <p>
           <Button small onPress={() => setConfirmTabulate(true)}>
-            Count my ballot
+            Accept Ballot As Is
```

Comment:
We're using `Count Ballot As Is` in other places. Let's make this consistent. I prefer count over accept.

Reply:
Can do… however in this case nothing can be "counted" because the document being scanned is not a ballot. Hence "accept" rather than "count".

Does this change your mind?

Reply:
Changing to "Cast Ballot As Is"